README.md

```diff
@@ -136,12 +140,6 @@ To verify that Secrets Store CSI Driver has started, run:
 Now you can follow these steps https://github.com/deislabs/secrets-store-csi-driver#use-the-secrets-store-csi-driver
 to create a SecretProviderClass resource, and a deployment using the SecretProviderClass.
 
-$ kubectl --namespace=dev get pods -l "app=secrets-store-csi-driver"
-NAME                                     READY     STATUS    RESTARTS   AGE
-csi-secrets-store-attacher-0                  1/1       Running   0          43s
-csi-secrets-store-secrets-store-csi-driver-f8lw6   2/2       Running   0          43s
-csi-secrets-store-secrets-store-csi-driver-hj445   2/2       Running   0          43s
-
 ```
 
 <details>
```

Comment:
I think we need to add steps here for installing provider using deployment yamls?